subprojects/soak/src/integTest/groovy/org/gradle/jvm/toolchain/JavaToolchainDownloadSoakTest.groovy

```diff
@@ -66,29 +76,7 @@ class JavaToolchainDownloadSoakTest extends AbstractIntegrationSpec {
 
         then:
         javaClassFile("Foo.class").assertExists()
-        assertJdkWasDownloaded(ECLIPSE_DISTRO_NAME)
-    }
-
-    def "can download missing j9 jdk automatically"() {
```

Comment:
❓ Was the removal of this test intentional? Why?

Reply:
I didn't feel like it was adding value. My thinking was that we should test that auto-downloading works in general and then testing that the specific requests are mapped to the right URL should be checked in tests of the resolver. The only thing this test was doing differently than the one still there is to use a different type of request. And that's tested in the Foojay resolver.